scripts/apps/vocabularies/controllers/VocabularyEditController.tsx

```diff
@@ -85,9 +100,9 @@ export function VocabularyEditController(
         if (angular.isDefined(response.data._issues)) {
             if (angular.isDefined(response.data._issues['validator exception'])) {
                 notify.error(gettext('Error: ' +
-                                     response.data._issues['validator exception']));
+                    response.data._issues['validator exception']));
```

Comment:
if you're touching this already - let's correct it
`gettext('Error: {{message}}', {message: response.data._issues['validator exception']})`